src/urllib3/util/request.py

```diff
@@ -21,7 +25,14 @@
 else:
     ACCEPT_ENCODING += ",br"
 
-_FAILEDTELL = object()
+
+class _TYPE_FAILEDTELL(Enum):
+    token = 0
+
+
+_FAILEDTELL: "Final[_TYPE_FAILEDTELL]" = _TYPE_FAILEDTELL.token
+
+_TYPE_BODY_POSITION = Optional[Union[int, _TYPE_FAILEDTELL]]
```

Comment:
Apologies for my earlier comment, I misspoke. This should be `Union[int, _TYPE_FAILEDTELL]`. This allows us to accurately use the type in places where it isn’t actually optional.